packaging/linux/openra.metainfo.xml.in

```diff
@@ -42,28 +43,15 @@
       <caption>Multiplayer lobby in the Tiberian Dawn mod</caption>
     </screenshot>
   </screenshots>
-  <url type="homepage">http://www.openra.net</url>
+  <url type="homepage">https://www.openra.net</url>
+  <url type="bugtracker">https://github.com/OpenRA/OpenRA/issues</url>
   <update_contact>paul_at_chote.net</update_contact>
-  <content_rating type="oars-1.0">
-    <content_attribute id="violence-cartoon">none</content_attribute>
-    <content_attribute id="violence-fantasy">none</content_attribute>
+  <content_rating type="oars-1.1">
```

Comment:
https://hughsie.github.io/oars/generate.html suggests that oars-1.1 is only supported by recent versions of Gnome Software. Do you know how older versions will handle this, or other software (is there any?) that consumes the metainfo?

Reply:
> supported by recent versions of Gnome Software

Gnome 3.27.3 was released over 3 years ago. Ubuntu 18.04 and Debian Stable ship newer versions.

Reply:
As both are defined in the same specification https://github.com/hughsie/oars/blob/master/specification/oars-1.1.md my guess is that this also works with KDE Discover and older versions of GNOME Software.